### DISCUSSION TITLE
Hello, Can we use saleor for my SaaS platform ?

### DISCUSSION THREAD

**Discussion by Spriteiplan** [saleor/saleor#9453] [General]
Hello, Can we use saleor for my SaaS platform ? If yes, Is it in open-source version or paid version ? Thanks

**Comment by karolkielecki** [comment]
Hi @Spriteiplan, please provide your information--a use case would be helpful to answer your question.

**Reply by moneyaddixt** [reply]
For e..g. installing saleor.io, editing the website and start adding subscription plans for users who would like to build their own eccomerce store.
